website/sidebars.js

```diff
@@ -403,6 +403,7 @@ const sidebars = {
             `build-apps/how-to-guides/navigate-between-pages`,
             `build-apps/how-to-guides/create-custom-nav-bar`,
             'build-apps/how-to-guides/Create-Custom-Widgets-Using-Iframe',
+            'build-apps/how-to-guides/custom-widget',
```

Comment:
remove it from sidebar for now. we will only publish this after we have reviewed it thoroughly.